evil-commands.el

```diff
@@ -636,6 +636,27 @@ and jump to the corresponding one."
           (when ov
             (goto-char (overlay-start ov))))))))
 
+(defun evil-ispell-mark-word-as-good (word)
+  "Add WORD under the cursor as a good word to the Ispell dictionary."
+  (interactive (list (thing-at-point 'word)))
+  (ispell-send-string (concat "*" word "\n"))
```

Comment:
`ispell-send-string` isn't autoloaded, so you must add `(require 'ispell)` somewhere, for example at the beginning of the file.

Reply:
done